addons/crm_livechat/models/chatbot_script_step.py

```diff
@@ -26,12 +38,11 @@ def _chatbot_crm_prepare_lead_values(self, discuss_channel, description):
 
     def _process_step(self, discuss_channel):
         self.ensure_one()
-
+        if self.step_type == "create_lead_and_forward":
```

Comment:
That feels strange to me, isn't it possible to be a bit less :ping_pong: style and call _process_step_create_lead then a subsequent "assign" step ? and/or add an option to _process_step_create_lead ? Current code is really ping-pong style: you don't call _process_step_create_lead because you call _process_step_create_lead_and_forward that manually calls _process_step_create_lead. And call to super is skipped, not sure why ? Well this is quite unclear :) .